packages/xo-server-auth-oidc/index.js

```diff
@@ -136,6 +140,30 @@ class AuthOidc {
     )
   }
 
+  // Synchronize user's groups.
+  async _synchronizeGroups(user, oidcGroups) {
+    if (Array.isArray(oidcGroups) && oidcGroups.length > 0) {
```

Comment:
Is there a way for `oidcGroups` to not be an array?
If yes, we should at least add a log to avoid "silent behavior"
Else, `Array.isArray` is not necessary